balloon.py

```diff
@@ -910,6 +910,122 @@ def get_wind(RapData,altitude):
 
     return (ve,vn)
 
+
+
+def getPositions(vz, lat, lon, alt, airspeed, head, querytime):
```

Comment:
not really sure why this function is here, since the inner function does most of the work anyway?

Reply:
The inner function is the one that extrapolates a position once. The outer function does it for a longer time (say 2 min) assuming the same wind speed, ascent rate, etc. So the outer function would return a dataframe of all positions within 2 min, while the inner function would return just a single array of balloon info